Dockerfile

```diff
@@ -1,11 +1,7 @@
-ARG PYTHON_VERSION=2.7
+ARG PYTHON_VERSION=3.7
```

Comment:
Shouldn't that be `3.6`?

Reply:
For this one, it's the version on which it will be compiled. Doesn't have a relationship with the minimal version